{
  if (!called) {
    throw new AssertionError();
  }
}
